NetCoreDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,311 @@
+﻿@model DevExtreme.NETCore.Demos.ViewModels.DropDownButtonViewModel
+
+<div>
+    <div class="widget-container">
+        @(Html.DevExtreme().Sortable()
+            .MinWidth(500)
+            .MinHeight(150)
+            .MaxHeight(370)
+            .Handles("right")
+            .Area(".widget-container")
+            .Content(
+                Html.DevExtreme().Toolbar()
+                    .ID("toolbar")
+                    .Multiline(true)
+                    .Items(items => {
+                        items
+                            .Add()
+                            .Widget(w => w
+                                .Button()
+                                .Icon("undo")
+                                .OnClick("undoButtonClick")
+                            )
+                            .Location(ToolbarItemLocation.Before);
+                        
+                        items
+                            .Add()
+                            .Widget(w => w
+                                .Button()
+                                .Icon("redo")
+                                .OnClick("redoButtonClick")
+                            )
+                            .Location(ToolbarItemLocation.Before);
+                        
+                        items
+                            .Add()
+                            .CssClass("toolbar-separator-container")
+                            .Template(new TemplateName("separator-template"))
+                            .MenuItemTemplate(new TemplateName("menu-separator-template"))
+                            .Location(ToolbarItemLocation.Before)
+                            .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+
+                        items
+                            .Add()
+                            .Widget(w => w
+                                .DropDownButton()
+                                .Width("100%")
+                                .DisplayExpr("Text")
+                                .KeyExpr("Size")
+                                .SelectedItemKey("14")
+                                .UseSelectMode(true)
+                                .DataSource(Model.FontSizes)
+                                .OnSelectionChanged("changeSize")
+                                .ItemTemplate(new JS("fontSizeItemTemplate"))
```

Comment:
Why don't we use asp `TemplateName` here as in neighbor places?

Reply:
Because we need use input params like 'item.text' for template rendering. And so code looks more cleaner then if we use ASP-template pattern in my opinion.